groovy/gradle.java/src/org/netbeans/modules/gradle/java/customizer/CompileOptionsPanel.java

```diff
@@ -171,7 +146,7 @@ private void initComponents() {
         lbCompileOnSave = new javax.swing.JLabel();
         cbAugmentedBuild = new javax.swing.JCheckBox();
         lbPlatform = new javax.swing.JLabel();
-        cbPlatform = new javax.swing.JComboBox<>();
+        cbPlatform = PlatformsCustomizer.createPlatformComboBox(new PlatformComboBoxModel());
```

Comment:
This usage could be replaced with the no-argument function.

Reply:
Done. Thanks!